src/Conversion/ONNXToStableHlo/ONNXToStableHloCommon.cpp

```diff
@@ -2,18 +2,19 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-//====----- ONNXToMhloCommon.cpp - ONNX dialects to Mhlo lowering ---------===//
+//====----- ONNXToStableHloCommon.cpp - ONNX dialects to StableHlo lowering
+//---------===//
```

Comment:
nit formatting; this happens also elsewhere, please do a sweep

Reply:
Thanks!  Fixed in the most recent commit.